pkg/util/span/frontier.go

```diff
@@ -50,14 +50,12 @@ type Frontier interface {
 	// but it will prevent frontier nodes from being efficiently re-used.
 	Release()
 
-	// Entries invokes the given callback with the current timestamp for each
-	// component span in the tracked span set.
-	// The fn may not mutate this frontier while iterating.
-	// TODO(yang): Delete this function and replace usages with All.
-	Entries(fn Operation)
-
-	// SpanEntries invokes op for each sub-span of the specified span with the
-	// timestamp as observed by this frontier.
+	// All returns an iterator over the entries in the frontier.
+	// Updates to the frontier are restricted until iteration is stopped.
+	All() iter.Seq2[roachpb.Span, hlc.Timestamp]
```

Comment:
why not keep the name `Entries()`?

Reply:
I was on the fence cuz the golang naming convention is to call the main iterator `All` but I changed it back so that we preserve the name